src/main/resources/js/actions/PanelActions.js

```diff
@@ -0,0 +1,13 @@
+import alt from '../alt';
```

Comment:
yeah this is the way to do it, it is pretty weird 

straight from the source:
http://alt.js.org/guide/actions/